sda-commons-web-autoconfigure/src/main/resources/org/sdase/commons/spring/boot/web/defaults.properties

```diff
@@ -6,3 +6,9 @@ server.error.whitelabel.enabled=false
 server.tomcat.basedir=${java.io.tmpdir}/tomcat
 
 request.body.max.size=1 MB
+
```

Comment:
Should this additional properties be moved to `tracing.properties`/`monitoring.properties`?